### ISSUE TITLE
Breaking change v1.0.0 - ibmcloud iam access-group-policy-create

### ISSUE THREAD

**Issue by jmgalvez** [IBM-Cloud/ibm-cloud-cli-release#111]
Hi there,

We are testing the new ibmcloud cli version 1.0.0.

We have noticed that it is behaving different when creating a new AG policy, as shown below:

```
$> ibmcloud iam access-group-policy-create AG1 --roles Administrator,Manager 
FAILED
Role Manager was not found. Valid roles are Viewer, Administrator, Operator, Editor.
```

In version 0.22.1 the very same command worked fine. That broke our automation, unfortunately.

Could you please confirm this point? 

If yes, could you please add it to the release notes as a breaking change?

We would really appreciate your feedback here.

Thank you in advance.

**Comment by steveclay** [comment]
The valid roles are defined here https://cloud.ibm.com/docs/account?topic=account-userroles . `Manager` is now a service role in IAM is why I believe you encountered this problem.

**Comment by jmgalvez** [comment]
Hi @steveclay,

Thank you for your response. This is quite old and not relevant for us anymore, we are using ibmcloud 2.0.3 now.

The command _ibmcloud iam access-group-policy-create AG1 --roles Administrator,Manager_ works perfectly on 2.0.3 and previous versions. 

We had that issue on April 2020, when suddenly our automation failed when we moved to version 1.0.0 from the previous 0.x.x versions, but it was fixed in a later update, don't remember exactly the version number.
